### ISSUE TITLE
AttributeError: 'module' object has no attribute 'pdtLocale_icu'

### ISSUE THREAD

**Issue by phoebebright** [bear/parsedatetime#212]
Get this error when running tests.  

<del>Was able to "fix" by reverting to 

    from .pdt_locales import load_locale

    pdtLocales = {
        'en_US': load_locale('en_US'),
        'en_AU': load_locale('en_AU'),
        ...
</del>
Fix only worked for some tests.

Using python 2.7.10

**Comment by idpaterson** [comment]
Hi Phoebe, would you please provide some context as to where that code was added, what it was originally, and in which version of parsedatetime you noticed the problem?

**Comment by phoebebright** [comment]
I'll delete this in a minute.  I have a fork which comments out the force addition of . and - to the date separators and in syncing with your current master I have clearly made a mess!  Much quicker to start with a clean fork and change one line of code!

**Comment by idpaterson** [comment]
No problem, parsedatetime version 3.0 is underway and once the new test suite is complete it will be time to move on to fixes like #152 based on your original request.

**Comment by phoebebright** [comment]
I look foward to it.  Can I make a request?

**Comment by idpaterson** [comment]
Certainly, it's a good time to report any bugs or feature requests. For 3.0 it's unlikely that any major new features will be added unless pull requests are submitted since there are a lot of bugs to fix, but if you have an idea go ahead and make a new issue to share it.

**Comment by phoebebright** [comment]
Will do.

**Comment by gdudek** [comment]
Workaround: pip install 'parsedatetime==2.5'
